website/content/en/v0.6.1/faq.md

```diff
@@ -29,7 +29,7 @@ Specify the desired architecture when you deploy workloads.
 
 ### What RBAC access is required?
 All of the required RBAC rules can be found in the helm chart template.
-See the [rbac.yaml](https://github.com/aws/karpenter/blob/main/charts/karpenter/templates/controller/rbac.yaml) file for details.
+See [clusterrolebinding.yaml](https://github.com/aws/karpenter/blob/main/charts/karpenter/templates/clusterrolebinding.yaml), [clusterrole.yaml](https://github.com/aws/karpenter/blob/main/charts/karpenter/templates/clusterrole.yaml), [rolebinding.yaml](https://github.com/aws/karpenter/blob/main/charts/karpenter/templates/rolebinding.yaml), and [role.yaml](https://github.com/aws/karpenter/blob/main/charts/karpenter/templates/role.yaml) files for details.
```

Comment:
these links are not accurate for v0.6.1 .  

You'll need to get links at v0.6.1 tag like this one: https://github.com/aws/karpenter/blob/8cca5e9a8b30cea911a032bd0141316fbfab0b2d/charts/karpenter/templates/controller/rbac.yaml 

The chart looks much different today than it did in v0.6.1 :D